src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/NeoForgeDataMaps.java

```diff
@@ -110,6 +128,19 @@ public class NeoForgeDataMaps {
     public static final DataMapType<GameEvent, VibrationFrequency> VIBRATION_FREQUENCIES = DataMapType.builder(
             id("vibration_frequencies"), Registries.GAME_EVENT, VibrationFrequency.CODEC).synced(VibrationFrequency.FREQUENCY_CODEC, false).build();
 
+    /**
+     * The {@linkplain Block} data map that replaces {@link HoneycombItem#WAXABLES}.
+     * <p>
+     * The location of this data map is {@code neoforge/data_maps/item/waxables.json}, and the values are objects with 1 field:
```

Comment:
The datamap is in `neoforge/data_maps/block/`, not in `neoforge/data_maps/item/`